src/LodeRunner.UI/src/components/Configs/index.js

```diff
@@ -142,19 +131,13 @@ const Configs = () => {
                 </div>
               </div>
               <div className="configs-item-options">
-                <button
-                  className="unset"
-                  type="button"
-                  onClick={openConfigFormModal(configId)}
-                  onKeyDown={openConfigFormModal(configId)}
-                  aria-label="Edit Load Test Config"
-                >
+                <A href={`/configs/${configId}`}>
```

Comment:
Should the anchor tag be lowercase \<a\/> ?

Reply:
A comes from: import { A } from "hookrouter";